src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+@import "./utils/_variables.scss";
+@import "./utils/_mixin.scss";
+
+.calendar {
+
+  display: flex;
+  justify-content: center;
+
+  &__box {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap-size;
+
+    width: $calendar-width;
+    padding-top: 40px;
+    padding-left: center;
+
+    @for $i from 1 through 31 {
```

Comment:
can we have month length 1? or it can't be down than 28?

Reply:
yes, there is a possibility

![image](https://user-images.githubusercontent.com/11179402/187665351-3e2d1d51-9de8-4392-af94-b2fd9e6278fd.png)